elytron/src/test/resources/org/wildfly/extension/elytron/tls-ibm.xml

```diff
@@ -149,7 +150,7 @@
             <client-ssl-context name="ClientSslContextOnlyTLS1" protocols="TLSv1" key-manager="ClientKeyManager"
                                 trust-manager="CaTrustManager" providers="ManagerProviderLoader" />
             <client-ssl-context name="ClientSslContextDefaultProtocols" key-manager="ClientKeyManager"
-                                trust-manager="CaTrustManager" providers="ManagerProviderLoader" />
+                                trust-manager="CaTrustManager" providers="ManagerProviderLoader" protocols="TLSv1 TLSv1.1 TLSv1.2"/>
```

Comment:
This change should also be added to the other `tls-*.xml` files to keep them all in sync. Since this `client-ssl-context` has been updated to explicitly configure protocols instead of just relying on the default protocols, I think it should be renamed from "ClientSslContextDefaultProtocols" to something else. It would also be good to double check that only the SSLv2Hello test is making use of this `client-ssl-context`. If another test was relying on the default protocol configuration, we should instead add a new `client-ssl-context` with the necessary protocols.